### PR TITLE
Document bitwise query operators

### LINKED ISSUE #1868: Add section for `bitwise query operators`
### What should be done?

Please see:

- #256
- #442

More details here:
https://www.mongodb.com/docs/v6.0/reference/operator/query-bitwise/

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #1868.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [x] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/reference/operators/bitwise-operators.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+The output:
+
+```js
+[
+  { _id: 1, value: 23, binaryValue: '10111' },
+  { _id: 4, value: 102, binaryValue: '1100110' }
+]
+```
+
+See the [$BitsallClear query operators](#bitsallclear) section for more usage examples.
```
Comment (w84thesun) [inline website/docs/reference/operators/bitwise-operators.md:104]:
```suggestion
See the [$bitsAllClear query operator](#bitsallclear) section for more usage examples.
```

File: website/docs/reference/operators/bitwise-operators.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+```js
+[
+  { _id: 2, value: 56, binaryValue: '111000' },
+  { _id: 3, value: 67, binaryValue: '1000011' },
+  { _id: 4, value: 102, binaryValue: '1100110' }
+]
+```
+
+See the [$BitsallClear query operators](#bitsallclear) section for more usage examples.
```
Comment (w84thesun) [inline website/docs/reference/operators/bitwise-operators.md:132]:
```suggestion
See the [$bitsAllClear query operator](#bitsallclear) section for more usage examples.
```

File: website/docs/reference/operators/bitwise-operators.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+```js
+[
+  { _id: 1, value: 23, binaryValue: '10111' },
+  { _id: 3, value: 67, binaryValue: '1000011' },
+  { _id: 4, value: 102, binaryValue: '1100110' },
+  { _id: 5, value: 5, binaryValue: '101' }
+]
+```
+
+See the [$BitsallClear query operators](#bitsallclear) section for more usage examples.
```
Comment (w84thesun) [inline website/docs/reference/operators/bitwise-operators.md:161]:
```suggestion
See the [$bitsAllClear query operator](#bitsallclear) section for more usage examples.
```

**Review by w84thesun — COMMENTED** [review]
Nice! Added a few small fixes.

**Review by ptrfarkas — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]

**Pull request merged**